typescript/infra/config/environments/mainnet3/owners.ts

```diff
@@ -202,6 +208,13 @@ export const icas: Partial<
   nero: '0xbBdb1682B2922C282b56DD716C29db5EFbdb5632',
   torus: '0xc1e20A0D78E79B94D71d4bDBC8FD0Af7c856Dd7A',
   xpla: '0x24832680dF0468967F413be1C83acfE24154F88D',
+
+  // Feb 3, 2025 batch
+  // ----------------------------------------------------------
+  glue: '0x24832680dF0468967F413be1C83acfE24154F88D',
+  matchain: '0x66af72e46b3e8DFc19992A2A88C05d9EEFE01ffB',
```

Comment:
couldn't check this because of an ica script error
```
Failed to process matchain: Error: missing revert data in call exception; Transaction reverted without a reason string
```

Reply:
yeah matchain RPC has been on fire the last 24 hours :(
![image](https://github.com/user-attachments/assets/330b96b9-03a9-45aa-b8fd-3eb5ed04c5e3)